eth/tracers/native/call_flat.go

```diff
@@ -248,6 +248,7 @@ func (t *flatCallTracer) GetResult() (json.RawMessage, error) {
 
 // Stop terminates execution of the tracer at the first opportune moment.
 func (t *flatCallTracer) Stop(err error) {
+	t.reason = err
```

Comment:
I'd go the other way..
remove t.reason here, and return t.tracer.reason from GetResults

Reply:
Infura has submitted a patch with `t.reason = err ` https://github.com/ethereum/go-ethereum/pull/29623/files should i still update as above or let it be as it is (cause it will cause conflict in future while merging geth, I don't have a preference either way)?